plugin.xml

```diff
@@ -48,7 +48,6 @@ xmlns:android="http://schemas.android.com/apk/res/android">
 		<resource-file src="src/android/cordova-plugin-firebase-strings.xml" target="res/values/cordova-plugin-firebase-strings.xml" />
 		<source-file src="src/android/FirebasePlugin.java" target-dir="src/org/apache/cordova/firebase" />
```

Comment:
I think you have to delete class FirebasePluginInstanceIDService from this file too

I talk about this lines

`<service android:name="org.apache.cordova.firebase.FirebasePluginInstanceIDService">
				<intent-filter>
					<action android:name="com.google.firebase.INSTANCE_ID_EVENT"/>
				</intent-filter>
			</service>`

Reply:
This is resolved by commit fed6a1e32a4d256fd498580200741bb5a14127ef